internal/cli/atlas/customdbroles/list.go

```diff
@@ -49,8 +49,7 @@ func (opts *ListOpts) initStore(ctx context.Context) func() error {
 }
 
 func (opts *ListOpts) Run() error {
-	listOpts := opts.NewListOptions()
-	r, err := opts.store.DatabaseRoles(opts.ConfigProjectID(), listOpts)
+	r, err := opts.store.DatabaseRoles(opts.ConfigProjectID())
```

Comment:
if this is no longer paginated can you please remove `cli.ListOpts` from `ListOpts`

Reply:
is the v2 no longer paginated or we do not support pagination on the v2 SDK?

Reply:
the manual docs documented this as paginated, turns out it doesn't have such things

Reply:
Updated to mark the flags as `deprecated`